Flow.Launcher.Plugin/Result.cs

```diff
@@ -140,9 +146,15 @@ public string PluginDirectory
             set
             {
                 _pluginDirectory = value;
-                if (!string.IsNullOrEmpty(IcoPath) && !Path.IsPathRooted(IcoPath))
+                    if (!string.IsNullOrEmpty(IcoPath) && !Path.IsPathRooted(IcoPath))
                 {
-                    IcoPath = Path.Combine(value, IcoPath);
```

Comment:
@taooceros @Garulf do you guys know why we are setting IcoPath while setting PluginDirectory?

Reply:
The original code was set up to add the plugin's root path to any relative paths. This would mangle any URI strings set in IcoPath because it would be treated as a relative path.

For example:
`http://test.org/file.zip` would become `c:\<pluginDir>\http://test.org/file.zip`